packages/config-loader/src/lib/cli.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AppConfig } from '@backstage/config';
+import { JsonObject } from '@backstage/types';
+
+export type CliConfigOptions = {
+  publicPath?: string;
+  backendUrl?: string;
+};
+
+/**
+ * Read specific parameters from the CLI and add them to the build config.
+ * @param opts CLI passed parameters.
+ * @returns Array of config, empty if there is no relevant passed in cli options.
+ *
+ * @public
+ */
+export function readCliConfig(opts?: CliConfigOptions): AppConfig[] {
```

Comment:
Let's not add this to `config-loader` and keep it in the CLI instead. This is a much lower level package where I wouldn't want to be doing these bits.